### PR TITLE
feat(build): #1126 retry makepythonpypienvironment

### PR DESCRIPTION
**Pull request by rohaquinlop**
- retry up to 3 times on curl errors.

### CODE REVIEW

**Comment by dsalaza4** [comment]
Beautiful approach!

Were you able to test it?

File: src/args/make-python-pypi-environment/default.nix [hunk 1]
```diff
@@ -104,6 +104,10 @@ assert builtins.any (_: _) [
             inherit name;
             inherit sha256;
             inherit url;
+            curlOptsList = [
+              "--retry 3"
```
Comment (dsalaza4) [inline src/args/make-python-pypi-environment/default.nix:108]:
```suggestion
              "--retry 
              "3"
```

Comment (dsalaza4) [reply src/args/make-python-pypi-environment/default.nix:108]: curl: option --retry 3: is unknown

**Comment by rohaquinlop** [comment]
> Beautiful approach!
> 
> Were you able to test it?

Yes, It runs local

File: src/args/make-python-pypi-environment/default.nix (hunk 1 above)
Comment (rohaquinlop) [reply src/args/make-python-pypi-environment/default.nix]: Done

**Comment by dsalaza4** [comment]
Adding curl opts to `nixpkgs.fetchurl` broke caches from cachix 🤔 

Maybe we'll have to clean caches when upgrading Universe.

**Review by dsalaza4 — APPROVED** [review]
lgtm

**Pull request merged**